app/main.ipynb

```diff
@@ -56,33 +57,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "py_array = [0 for _ in range(1001)]\n",
+    "sys.getsizeof(py_array)"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T12:02:41.673421Z",
-     "start_time": "2023-07-03T12:02:41.666277Z"
+     "end_time": "2024-10-28T21:40:35.359699Z",
+     "start_time": "2024-10-28T21:40:35.352723Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "8856"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 6
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "outputs": [],
    "source": [
-    "# write your code here (for numpy syntax)"
+    "np_array = np.array(np.arange(1001), dtype=np.int16)\n",
```

Comment:
The numpy array `np_array` is created with values from 0 to 1000, but the task specifies that it should contain zeros. Use `np.zeros(1000, dtype=np.int16)` to create an array of zeros with minimal memory usage.